src/main/java/gregtech/common/covers/filter/SimpleItemFilter.java

```diff
@@ -82,38 +65,95 @@ public boolean showGlobalTransferLimitSlider() {
     }
 
     @Override
-    public int getTotalOccupiedHeight() {
-        return 36;
-    }
-
-    @Override
-    public void initUI(Consumer<Widget> widgetGroup) {
+    public void initUI(Consumer<gregtech.api.gui.Widget> widgetGroup) {
```

Comment:
Do we still need to do anything here? Since the filters should have been moved to new implementation, we don't need to keep creating the old implementation.

Reply:
Currently, the advanced item/fluid detectors are not updated to new MUI, so these methods are still needed for the filter's UI to show up. Once the advanced detectors are ported to MUI, these can be removed.